scene/gui/line_edit.cpp

```diff
@@ -1200,7 +1213,7 @@ void LineEdit::_reset_caret_blink_timer() {
 
 void LineEdit::_toggle_draw_caret() {
 	draw_caret = !draw_caret;
-	if (is_visible_in_tree() && has_focus() && window_has_focus) {
+	if (is_visible_in_tree() && (has_focus() || caret_force_displayed) && window_has_focus) {
```

Comment:
I think this should be something like the following, else it will not toggle when the window loses focus:

```suggestion
	if (is_visible_in_tree() && ((has_focus() && window_has_focus) || caret_force_displayed) {
```